src/net/sf/mpxj/mspdi/schema/Project.java

```diff
@@ -11,13 +11,15 @@
 import java.math.BigInteger;
 import java.util.ArrayList;
 import java.util.List;
+
```

Comment:
I've updated my copy of the `xsd` file to include the correct data type and regenerated the JAXB schema.